org.mixedrealitytoolkit.spatialmanipulation/Constraints/MinMaxScaleConstraint.cs

```diff
@@ -82,7 +82,12 @@ public override void ApplyConstraint(ref MixedRealityTransform transform)
                 max = InitialWorldPose.Scale.Mul(max);
             }
 
-            transform.Scale = Vector3.Max(min, Vector3.Min(max, transform.Scale));
+            // Apply the min and max constraints per axis
+            transform.Scale = new Vector3(
+                Mathf.Clamp(transform.Scale.x, min.x, max.x),
+                Mathf.Clamp(transform.Scale.y, min.y, max.y),
+                Mathf.Clamp(transform.Scale.z, min.z, max.z)
+            );
```

Comment:
Perhaps I'm missing something, but Vector3.Max and Vector3.Min already do a component-wise comparison for x, y, and z, individually between the min and max matching components.  How does this change alter the result?

Reply:
@whebertML you're totally right, I just tested it! My bad, for some reason I thought it wasn't working, one of those late nights coding missteps sorry about that.. disregard the PR

Reply:
@LaneMax Thanks for being proactive in opening it in the first place! I've closed it now, per your "disregard" comment. But looking forward to the next one 🙂

Reply:
Thx again @whebertML & @keveleigh! Yeah was one of those late night coding sessions working on network transform scaling 🤯